frameworks/g-cloud-9/questions/services/publicSectorNetworksTypes.yml

```diff
@@ -15,13 +15,19 @@ type: checkboxes
 options:
   - label: Public Services Network (PSN)
     value: psn
+    filter_label: Public Services Network (PSN)
```

Comment:
I'm going to remove the ones like this where they are exactly the same. Am going to leave lower-case variants for now, though.